### ISSUE TITLE
Metrics computation during the training

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#23]
Hi @alykhantejani 

Do you plan to integrate some metrics computation in the trainer or this should be done on the user side with event handlers (and logging in the same way as ignite) ?
Anyway, it would be good to display actually computed loss values.

**Comment by alykhantejani** [comment]
Hi @vfdev-5,

I plan to have metric computation available as handlers (probably under `ignite.handlers.metrics`) which users can then attach to events. 

These can take a `logger` argument, which could default to `print` but the user can pass in `my_logger.info` instead for example and the handler will just call `logger(result)`

As always PRs are welcome :)

**Comment by alykhantejani** [comment]
@vfdev-5 some basic metrics have now been merged. Please open an issue for any specific metric request you may have

**Comment by sjsy** [comment]
@alykhantejani
Hi, 
I am newbie in pythorch. Is there a simple example that uses some metrics?
Thanks

**Comment by vfdev-5** [comment]
@sjsy, yes, see [here](https://github.com/pytorch/ignite/blob/master/examples/mnist.py#L56), for example.

**Comment by sjsy** [comment]
@vfdev-5 Thanks a lot!

**Comment by sjsy** [comment]
@vfdev-5 , in the example you pointed out, the metrics are attached to create_supervised_evaluator. How can I attach some metrics to create_supervised_trainer? So I can investigate training progress by attached metrics as well as the loss.
Thanks.

**Comment by vfdev-5** [comment]
@sjsy actually, you can do this like [this](https://github.com/vfdev-5/ignite-examples/blob/master/classification/cifar_10/cifar10_train_playground.py#L132) and [this](https://github.com/vfdev-5/ignite-examples/blob/master/classification/cifar_10/cifar10_train_playground.py#L287). HTH
